packages/web/src/components/ManageSocials/FbAuth.test.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+describe('components/FbAuth', () => {
+  const facebookLoginBtnTestId = 'facebookLoginBtn';
+  const getFbAuthAPI = response => ({
+    appId,
+    fields,
+    scope,
+    icon,
+    callback,
+    isDisabled,
+    size,
+    textButton,
+  }) => (
+    <div>
+      <div>{appId}</div>
+      <div>{fields}</div>
+      <div>{scope}</div>
+      <div>{icon}</div>
+      <div>{size}</div>
+      <button
+        data-testid={facebookLoginBtnTestId}
+        disabled={isDisabled}
+        onClick={callback.bind(null, response)}
+      >
+        {textButton}
+      </button>
+    </div>
+  );
+
+  const setUpMocks = ({
+    onResponseAction,
+    onErrorAction,
+    errorMessage,
+    buttonText,
+    response = { accessToken: '' },
+    isLoading = false,
+  }) => {
+    const mockDispatch = jest.fn();
+    jest.doMock('react-redux', () => ({
+      useSelector: () => ({ isLoading }),
+      useDispatch: () => mockDispatch,
+    }));
+
+    jest.doMock('react-facebook-login', () => getFbAuthAPI(response));
+
+    const { FbAuth } = require('./FbAuth');
+    const { container, getByTestId } = render(
+      <FbAuth
+        onResponseAction={onResponseAction}
+        onErrorAction={onErrorAction}
+        errorMessage={errorMessage}
+        buttonText={buttonText}
+      />
+    );
+    return { container, mockDispatch, getByTestId };
+  };
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const onResponseAction = 'NOICE_ALL_WENT_WELL';
+  const onErrorAction = 'ALARM_SOME_PROBLEM_HAPPENED';
+  const errorMessage = 'Chuck Norris has been defeated by Bruce Lee';
```

Comment:
Why is that an error though? The error would be the other way around!

Reply:
I am not quite sure about the question. But the thing that does happen in this place of the file is mocking of the FB auth button API